### PR TITLE
doc(distro): okd does not support ARM

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#8359, okd-project/okd#1165

### CODE REVIEW

File: content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ weight: 4
 
 To deploy Longhorn on a cluster provisioned with OpenShift 4.x, some additional configurations are required.
 
+**NOTE**: OKD does not support ARM platform yet. [[OKD website](https://www.okd.io/crc/?h=arm), [OKD in ARM platform github issue](https://github.com/okd-project/okd/issues/1165)]
```
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md:8]:
```suggestion
> **Note**: OKD currently does not support the ARM platform. For more information, see the [OKD website](https://www.okd.io/crc/?h=arm) and [GitHub issue #1165](https://github.com/okd-project/okd/issues/1165) (*OKD in ARM platform*).
```

**Review by jillian-maroket — COMMENTED** [review]
@mantissahz Let me know if you have concerns about the changes.

File: content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md (hunk 1 above)
Comment (mantissahz) [reply content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md]: Modified for all versions after v1.6.0. Thanks.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by innobead — APPROVED** [review]

**Pull request merged**